mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -140,3 +141,94 @@ def __repr__(self):
         """Compute the string representation."""
         return (f'{self.__class__.__name__}('
                 f'keys={self.keys}, meta_keys={self.meta_keys})')
+
+
+@PIPELINES.register_module()
+class HideAndSeek:
+    """Augmentation by informantion dropping in Hide-and-Seek paradigm. Paper
+    ref: Huang et al. AID: Pushing the Performance Boundary of Human Pose
+    Estimation with Information Dropping Augmentation (arXiv:2008.07139 2020).
+
+    Args:
+        prob_has (float): Probability of performing hide-and-seek.
+        prob_has_hide (float): Probability of hiding patches.
+        grid_sizes (list): List of optional grid sizes.
+    """
+
+    def __init__(self,
+                 prob_has=1.0,
+                 prob_has_hide=0.5,
```

Comment:
`prob_has_hide` -> `prob_hiding_patches`?

Reply:
'prob_hiding_patches' is better